controllers/factory/k8stools/expansion.go

```diff
@@ -167,8 +188,7 @@ func isStorageClassExpandable(ctx context.Context, rclient client.Client, pvc *c
 	}
 	// fast path at single namespace mode, listing storage classes is disabled
 	if !config.IsClusterWideAccessAllowed() {
-		log.Info("cannot detect if storageClass expandable at single namespace mode, expand PVC manually or enforce resizing with annotation to true", "pvc annotation", victoriametricsv1beta1.PVCExpandableLabel)
-		return false, nil
+		return false, fmt.Errorf("cannot detect if storageClass expandable at single namespace mode, need to expand PVC manually or enforce resizing by adding annotation `%s=true`", victoriametricsv1beta1.PVCExpandableLabel)
```

Comment:
I think, it's not an error. Since, in single namespace mode it'd be not possible to use PVC. Operator will produce following error each time and cannot reconcile resources without user action.

 I think, it's better to return false, nil.

Reply:
>Operator will produce following error each time

It won't be there each time, since sts will be updated for the first time and won't be marked as [isRecreated](https://github.com/VictoriaMetrics/operator/blob/fix-grow-sts-pvc/controllers/factory/k8stools/sts.go#L82), thus skip the pvc check.

I thought we should treat it as same as `failed to list storageClass` cause both of them can't list sc. In single namespace mode, I assume user should notice this error and fix it according to the notes.

Reply:
I's a breaking change for current single namespace mode users. I suggest to mention it at our docs + add an example how to configure vmsingle/vmcluster with manual annotation with storage class params

Reply:
you can update docs with additional PR.